### PR TITLE
Issue 1 readme

### PR DESCRIPTION
**Pull request by ZdenkaZ**
@hovi please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  - while working on it, Zdenka is learning to code in JavaScript
+  - Zdenka is learning HTML
+  - learning to use GitHub
+  - futher new exciting features to learn are comming up
+ 
+      >Totally indispensable is the infinite selfless help of Kaja (@hovi).
+ 
+Guess Numbers uses the basic principles of *cristal clean magic*. To divine a number heavenly **luck** is necessary. The approximate  description of conditions required to obtain this luck is describes the science of **astrology**. Guess with ease and pleasure on your  heart, because luck is writen in the *stars*.
+
+![A picture of zodiac signs.](https://time.com/5315377/are-zodiac-signs-real-astrology-history/)
```
Comment (hovi) [inline README.md]: You are not linking image, but an article. Make sure, you have correct URL, maybe you meant this?
https://api.time.com/wp-content/uploads/2018/06/zodiac-history-astrology-signs.jpg

File: README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  - Zdenka se při její tvorbě učí kódovat v JavaScriptu
+  - seznamuje se s HTML
+  - učí se zacházet s GitHubem
+  - více nových zajímavostí se jistě ještě ukáže
+    
+    > Nepostradatelná je při Zdenčině učení Kájova neutuchající velkorysá pomoc.
+    
+Hádání čísel používá základní aspekty *magie*. K uhádnutí čísla je potřeba **štěstí**, které padá z nebe, a k jeho vysvětlení se blíží **astrologie**.  Hádejte tedy vesele, protože štěstí je ve *hvězdách*.  
+
+![A picture of zodiac signs.](https://time.com/5315377/are-zodiac-signs-real-astrology-history/)
```
Comment (hovi) [inline README.md]: Same as above.

**Review by hovi — CHANGES_REQUESTED** [review]
Almost perfect, just change the image url.

File: README.md (hunk 1 above)
Comment (ZdenkaZ) [reply README.md]: Oh yeah. Check it out now.

**Review by hovi — APPROVED** [review]
First PR done!

File: README.md (hunk 1 above)
Comment (ZdenkaZ) [reply README.md]: @hovi

**Pull request merged**